README.md

```diff
@@ -32,16 +32,20 @@ Google Groups, IRC #activeadmin and Gitter are not actively monitored.
 
 ## Want to contribute?
```

Comment:
Isn't "support" a transitive verb so it would need a noun after, like "support _us_"? Or does this mean "support" as a noun (meaning something like... "help").

Reply:
Yes you are right. It means more like "help" in this regard, although I agree it would be better with "support us" instead.

The reason I suggested the heading change, is that to me "contribute" is used in OSS when discussing code or project changes while the latter is used with the financial side so I wanted to include both, although its not necessary.

Reply:
Maybe it makes sense to go back to the previous version here, then: two separate sections called "Want to contribute?" and "Want to support us?".

Reply:
Sure, that sounds good to me. 👍🏻

Reply:
Updated via https://github.com/activeadmin/activeadmin/pull/5649/commits/1e1ad9162da119fd0afcdf1b6795076e95be0ccf, this is ready for another pass! :+1: